units_cpp/src/units/headers/units/constants/planetary_rotations.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * planetary_rotations.h
+ *
+ *  Created on: Nov 3, 2017
+ *      Author: oriash93
+ */
+
+#ifndef SRC_UNITS_HEADERS_UNITS_CONSTANTS_PLANETARY_ROTATIONS_H_
+#define SRC_UNITS_HEADERS_UNITS_CONSTANTS_PLANETARY_ROTATIONS_H_
+
+#include <units/duration.h>
+
+namespace units {
+namespace planetary_rotations
```

Comment:
maybe "planetary_rotation_periods", since this is how u named the constants?

Reply:
you're right. will rename it